patches/0098-libavutil-hwcontext_vulkan-Add-one_memory-flag-to-ma.patch

```diff
@@ -0,0 +1,58 @@
+From f01d3a861bfa72d925b2afd2ece3a077ac9a3c3c Mon Sep 17 00:00:00 2001
+From: Wenbin Chen <wenbin.chen@intel.com>
+Date: Mon, 31 May 2021 13:36:35 +0800
+Subject: [PATCH 06/13] libavutil/hwcontext_vulkan: Add one_memory flag to make
+ vulkan compatible with vaapi device.
+
+Vaapi can import external surface, but all the planes of the external
+frames should be in the same drm object. A new flag is introduced and
+vulkan can choose to allocate planes in one memory according this flag.
+This flag will be enabled when the vulkan device is derived from vaapi
+device, so that this change will not affect current vulkan behaviour.
+
+Signed-off-by: Wenbin Chen <wenbin.chen@intel.com>
+---
+ libavutil/hwcontext_vulkan.c | 12 +++++++++++-
+ 1 file changed, 11 insertions(+), 1 deletion(-)
+
+diff --git a/libavutil/hwcontext_vulkan.c b/libavutil/hwcontext_vulkan.c
+index 9a29267aed..6417f59d4a 100644
+--- a/libavutil/hwcontext_vulkan.c
++++ b/libavutil/hwcontext_vulkan.c
+@@ -211,6 +211,9 @@ typedef struct VulkanDevicePriv {
+     /* Settings */
+     int use_linear_images;
+ 
++    /* map all planes to one memory */
++    int use_one_memory;
```

Comment:
This is a flag for memory allocation, not map, right ?

Reply:
This flag affect both allocation and map.